### ISSUE TITLE
Source Code Release Request

### ISSUE THREAD

**Issue by Anywayany** [Yu-Wang-0801/DCTA#1]
This is a good job, when will you release the source code? I'm curious to know how the Affine layer is implemented in ResMLP, thanks.

**Comment by Yu-Wang-0801** [comment]
Thank you for your attention! The code implementation of the Affine layer is as follows:
class Aff(nn.Module):
     def __init__(self, dim):
         super().__init__()

         self.alpha = nn.Parameter(torch.ones([1, 1, dim]))
         self.beta = nn.Parameter(torch.zeros([1, 1, dim]))

     def forward(self, x):
         x = x * self.alpha + self.beta
         return x
If this code is useful to you, please cite this paper, thank you!
